src/views/Game/GameTimings.tsx

```diff
@@ -1,27 +1,29 @@
 /*
  * Copyright (C)  Online-Go.com
  *
- * This program is free software: you can redistribute it and/or modify
```

Comment:
Gotta turn on the case-sensitivity 😄

Reply:
That's one way of looking at it, though in reality the problem was I have regex match turned on but I said `this.` 🤦‍♀️